DetectorDescription/DDCMS/interface/Filter.h

```diff
@@ -28,7 +28,7 @@ namespace cms {
     std::vector<std::string_view> keys;
     std::unique_ptr<Filter> next;
     struct Filter* up;
-    const DDSpecPar* spec;
+    const DDSpecPar* spec = {};
```

Comment:
```suggestion
    const DDSpecPar* spec = nullptr;
```
would seem to me to be easier to understand.